### DISCUSSION TITLE
Apply filter in APP from uri elements

### DISCUSSION THREAD

**Discussion by neopk** [directus/directus#19055] [Feature Requests]
### Summary

It's possible to filter elements by a button in app, nevertheless it's not possible to get a link in order access to a prefiltered collection.

### Basic Example

/admin/content/<collection>?label=eq.myPrefilteredTest which display the collection with the elements

### Motivation

The case is to communicate with other people on slack et give them all elements which are interresting.

### Detailed Design

Let's look basic example, this feature is quite simple to implement.

### Must Have:
- Filter with equals

### Drawbacks

Many case of filter to implement, maybe need to focus on equals and not equals in a first feature to have data if people use it.

### Alternatives

No way to provide link, user desn't want to apply filter manually when some tell them "i give you an url to directus, you have to filter on these and these elements;"

### Adoption Strategy

This proposal doesn't affect actual use of directus.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
